repositories/src/main/scala/cromwell/pipeline/datastorage/dto/WomToolNode.scala

```diff
@@ -0,0 +1,45 @@
+package cromwell.pipeline.datastorage.dto
+
+import play.api.libs.json.{ Format, JsNull, JsResult, JsSuccess, JsValue, Json, Writes }
+
+final case class WomToolNode(name: String, value: Option[String])
+
+object WomToolNode {
+  import OptionFormat._
+
+  implicit object WomToolNodeFormat extends Format[WomToolNode] {
+    override def writes(o: WomToolNode): JsValue = Json.obj(o.name -> o.value)
+
+    override def reads(json: JsValue): JsResult[WomToolNode] = {
+      val (name, value) = json.as[(String, Option[String])]
+      JsSuccess(WomToolNode(name, value))
+    }
+  }
+}
+
+final case class WomToolNodesList(nodes: List[WomToolNode])
+
+object WomToolNodesList {
+
+  implicit object WomToolNodesListFormat extends Format[WomToolNodesList] {
+    override def writes(o: WomToolNodesList): JsValue =
+      Json.obj(o.nodes.map(node => node.name -> Json.toJsFieldJsValueWrapper(node.value)): _*)
+
+    override def reads(json: JsValue): JsResult[WomToolNodesList] = JsSuccess(
+      WomToolNodesList(json.as[List[WomToolNode]])
+    )
+  }
+}
```

Comment:
I read your comment about necessity of such formatters for Cromwell.
I tried to take a look at how you are using it, but I failed to find any usage of these classes. If they are unused, why do you need to include them in this PR? :)

Reply:
Yes, I know about it. I will add endpoint for getting FileParameters in Cromwell-like format